gorb/gorb.go

```diff
@@ -122,9 +125,10 @@ func New(c *Config) (controller.Updater, error) {
 		backendDefinitions = append(backendDefinitions, backendDefinition)
 	}
 
-	var httpClient = &http.Client{
-		Timeout: time.Second * 5,
-	}
+	httpClient := pester.New()
+	httpClient.Timeout = c.HTTPClientTimeout
+	httpClient.MaxRetries = c.HTTPClientMaxRetries
+	httpClient.Backoff = pester.ExponentialBackoff
```

Comment:
May be better to not use a backoff, since we don't want this to retry for very long. Exponential would make more sense if we wanted to retry for a long period. I suppose if we used a max timeout of 60s or something exponential makes sense.

Reply:
have reverted back to the default backoff of 1s